### PR TITLE
Blueshield, but it's an ERT

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Different take on https://github.com/BeeStation/BeeStation-Hornet/pull/2553.

## Why It's Good For The Game

Emergency response team for when things are a BIT out of control, but does not warrant a full ERT. The blueshield is here to protect heads of Staff and Command from crumbling.
Emergency situations that require ERT could be:
* Revolution lasting more than a hour and heads have bunkered down for too long (tiebreaker?)
* Blood brother killed 3/5 command members and captain used console to ask help from CC
* Traitor got cocky and killed all security/most of command
* Captain is dead and HoP requested Blueshield
* VIP is in perma for whatever reason

## Changelog
:cl:
add: Blueshield, callable by admins
/:cl:

### CODE REVIEW

**Comment by VictorVow** [comment]
I like it, slightly more robust and serious than interns but not as OP as ERT.

**Comment by ghost** [comment]
The energy revolver they get is esentially a reskinned mini-egun, perhaps you should port this too?https://github.com/OracleStation/OracleStation/pull/9

Also i'm pretty sure they don't get any head protection, this PR gives them a moderatley armored beret https://github.com/OracleStation/OracleStation/pull/949

**Comment by eeSPee** [comment]
Code is done and working, but maybe it would be a good idea to have a spriter port in the sprites and whatnot?

Copying files from TG seems like that they might overwrite the already existing bee files and remove bee sprites/outfits.

**Comment by VictorVow** [comment]
@eeSPee we can discuss the workflow for how to port sprites on Discord

**Comment by eeSPee** [comment]
> @eeSPee we can discuss the workflow for how to port sprites on Discord

sure

File: code/game/objects/items/miscellaneous.dm [hunk 1]
```diff
@@ -306,3 +306,28 @@
...
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "blu-wrapped"
+	var/list/options = list(".38 Mars Special Revolver" = /obj/item/storage/box/blueshield/revolver,
+	".45 Enforcer Semi Automatic Pistol" = /obj/item/storage/box/blueshield/enforcer,
+	"Aegis SG7 Laser Gun" = /obj/item/storage/box/blueshield/laser)
+
+/obj/item/blueshield_gun_package/attack_self(mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/H = user
```
Comment (ike709) [inline code/game/objects/items/miscellaneous.dm:323]:
```suggestion
/obj/item/blueshield_gun_package/attack_self(mob/living/carbon/human/user)
	if(!istype(user))
		return
```

Comment (ike709) [reply code/game/objects/items/miscellaneous.dm:323]: Then replace `H` with `user` later on.

File: code/modules/projectiles/boxes_magazines/external/pistol.dm [hunk 2]
```diff
@@ -57,3 +57,24 @@
...
+	caliber = ".45"
+
+/obj/item/ammo_box/magazine/enforcer/update_icon()
+	icon_state = "enforcer-[stored_ammo.len]"
+
+/obj/item/ammo_box/magazine/enforcer/examine(mob/user)
+	..()
+	if(stored_ammo)
+		var/obj/item/ammo_casing/A = stored_ammo[stored_ammo.len]
+		to_chat(user, "TThere's a [A.name] at the top.")//it's either rubber or lead. better to know the difference
```
Comment (ike709) [inline code/modules/projectiles/boxes_magazines/external/pistol.dm:77]:
```suggestion
		to_chat(user, "There's a [A.name] bullet at the top.")//it's either rubber or lead. better to know the difference
```

**Review by ike709 — CHANGES_REQUESTED** [review]
Lots of unrelated icon changes.

**Comment by eeSPee** [comment]
What did I fuck up?

File: beestation.dme [hunk 3]
```diff
@@ -269,7 +269,6 @@
 #include "code\controllers\subsystem\parallax.dm"
 #include "code\controllers\subsystem\pathfinder.dm"
 #include "code\controllers\subsystem\persistence.dm"
-#include "code\controllers\subsystem\ping.dm"
```
Comment (ike709) [inline beestation.dme:272]: The hell is going on with this file?

Comment (PowerfulBacon) [reply beestation.dme:272]: That file shouldn't even exist on the repo

**Pull request closed without merging**